contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2313,40 +2314,6 @@ class tsqlBuilder : public tsqlCommonMutator
 		// TO-DO
 	}
 
-	void exitPredicate(TSqlParser::PredicateContext *ctx) override
```

Comment:
Just for double check, the real implement is not changed ? I read one by one , and didn't see any difference in implement.

Reply:
Some statements are handled by tsqlBuilder (like top-level SELECT statements in a batch), others by tsqlMutator (like such statements in a procedure body). this is a consequence of the structure of this code has ended up. Therefore some things need to be executed in both classes. This is not new.